package.json

```diff
@@ -13,6 +13,7 @@
     "i18next-browser-languagedetector": "^7.1.0",
     "i18next-http-backend": "^2.2.1",
     "lodash": "^4.17.21",
+    "npm": "^10.0.0",
```

Comment:
We don't need it here, npm should be installed globally

Reply:
At first, I thought it was necessary when I had issues with my git bash, but now i uninstalled it from the package.json